lib/node/runner/jsonrpc_test.go

```diff
@@ -0,0 +1,264 @@
+package runner
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+
+	rpcjson "github.com/gorilla/rpc/json"
```

Comment:
In the other file the import is name `jsonrpc`. I don't mind which one you choose but we should be consistent.

Reply:
@Geod24 I will fix it.